mavros/include/mavros/mavros_uas.h

```diff
@@ -65,17 +65,20 @@ namespace mavros {
  */
 class UAS {
 public:
-	using ConnectionCb = std::function<void(bool)>;
-	using lock_guard = std::lock_guard<std::recursive_mutex>;
-	using unique_lock = std::unique_lock<std::recursive_mutex>;
-
 	// common enums used by UAS
 	using MAV_TYPE = mavlink::common::MAV_TYPE;
 	using MAV_AUTOPILOT = mavlink::common::MAV_AUTOPILOT;
 	using MAV_MODE_FLAG = mavlink::common::MAV_MODE_FLAG;
 	using MAV_STATE = mavlink::common::MAV_STATE;
+	using MAV_CAP = mavlink::common::MAV_PROTOCOL_CAPABILITY;
 	using timesync_mode = utils::timesync_mode;
 
+	// other UAS aliases
+	using ConnectionCb = std::function<void(bool)>;
+	using CapabilitiesCb = std::function<void(MAV_CAP)>;
```

Comment:
I have the capabilities callback take a MAV_PROTOCOL_CAPABILITY enum in order to increase visibility of where people can find the cpp enum to use in their code. After adding the has_capability / has_capabilities functions to UAS I don't think this is needed any longer and it could be a void func(); What do you think?

Reply:
Leave it with parameter, that would be more explicit.